website/src/pages/Micro/Micro.jsx

```diff
@@ -0,0 +1,181 @@
+                        {dataRefs.map((ref, index) => {
+                            return (
+                                <React.Fragment key={index}>
+                                    <figure className='dropDown--line'></figure>
+                                    <span className='dropDown__ref' onClick={() => valueDropDown(ref, setGitRefRight, setCommitHashRight, setOpenDropDownRight)}>{ref.Name}</span>
+                                </React.Fragment>
+                            )
+                        })}
+                    </figure>
+                </div>
+                {error ? (
+                    <div className='apiError'>{error}</div> 
+                ) : (
+                isLoading ? (
+              <div className='loadingSpinner'>
+                <RingLoader loading={isLoading} color='#E77002' size={300}/>
+                </div>
+                ): ( 
+                <div className='micro__container'>
+                    <div className='micro__thead space--between'>
+                        <span className='width--12em'>Package</span>
+                        <span className='width--14em'>Benchmark Name</span>
+                        <span className='width--18em hiddenMobile'>Number of Iterations</span>
+                        <span className='width--18em hiddenTablet'>Time/op</span>
+                        <span className='width--6em'>More</span>
+                    </div>
+                    <figure className='micro__thead__line'></figure>
+                    <div className='space--between--flex data__top hiddenMobile'>
+                        <div className='width--12em'></div>
+                        <div className='width--14em'></div>
+                        <div className='width--18em space--between--flex'>
+                            <span className='width--100'>{gitRefLeft}</span> 
+                            <span className='width--100'>{gitRefRight}</span>
+                            <span className='width--100'>Diff %</span>
+                        </div>
+                        <div className='width--18em space--between--flex hiddenTablet'>
+                            <span className='width--100'>{gitRefLeft}</span> 
+                            <span className='width--100'>{gitRefRight}</span>
+                            <span className='width--100'>Diff %</span>
+                        </div>
+                        <div className='width--6em'></div>
+                    </div>
+                    {dataMicrobench.map((micro, index) => {
+                        const isEvenIndex = index % 2 === 0;
+                        const backgroundGrey = isEvenIndex ? 'grey--background' : '';
+                        return(
+                            <Microbench data={micro} key={index} className={backgroundGrey} gitRefLeft={gitRefLeft} gitRefRight={gitRefRight}/>
+                        )
+                    })}
```

Comment:
We still render data if the list of microbenchmark is empty:
![image](https://github.com/vitessio/arewefastyet/assets/35779988/f95ee364-3514-402a-9526-827a100de357)

We should check the first element of the list and decide to not render anything if the first element is empty.